ArduPlane/tiltrotor.cpp

```diff
@@ -309,7 +309,14 @@ void Tiltrotor::continuous_update(void)
         // Q_TILT_MAX. Anything above 50% throttle gets
         // Q_TILT_MAX. Below 50% throttle we decrease linearly. This
         // relies heavily on Q_VFWD_GAIN being set appropriately.
-       float settilt = constrain_float((SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)-MAX(plane.aparm.throttle_min.get(),0)) * 0.02, 0, 1);
+       const float throttle = SRV_Channels::get_output_scaled(SRV_Channel::k_throttle);
+       float settilt = 0.0f;
+       if (throttle > 0) {
+            // Remove throttle min
+            settilt = constrain_float((throttle - MAX(plane.aparm.throttle_min.get(),0)) * 0.02, 0, 1);
+       } else if (quadplane.fwd_thr_min < 0) {
+           settilt = constrain_float(throttle * 0.02, -1, 1);
+       }
```

Comment:
I think this has to be based on the param only. Moving between one and the other based on throttle means you get a step as we subtract plane min throttle or not. 

```suggestion
    if (quadplane.fwd_thr_min < 0) {
        settilt = constrain_float(throttle * 0.02, -1, 1);
    } else {
        // Remove throttle min
        settilt = constrain_float((throttle - MAX(plane.aparm.throttle_min.get(),0)) * 0.02, 0, 1);
    }
```